assets/sass/custom.scss

```diff
@@ -8,3 +8,13 @@
     }
   }
 }
+
+.card {
```

Comment:
I'm adding custom sass files to encapsulate styles, instead of adding them directly to `custom.scss`, for example, for the fixtures tables, I've added `_fixture_tables.scss` and added the styles in there, then imported the file in `custom.scss`

I believe this is one step forward if we want to move to a component based architecture in the frontend later.

Thoughts?